tests/test-metadata.php

```diff
@@ -1,8 +1,10 @@
 <?php
 
+use DMS\PHPUnitExtensions\ArraySubset\ArraySubsetAsserts;
+
 class MetadataTest extends \WP_UnitTestCase {
 
-	use utilsTrait;
+	use utilsTrait, ArraySubsetAsserts;
```

Comment:
This assertion was removed in PHPUnit 8 I needed to add an external package to use it `https://packagist.org/packages/dms/phpunit-arraysubset-asserts`